cli/src/ai/render-markdown.ts

```
@@ -0,0 +1,85 @@
+}
+
+function renderInline(line: string): string {
+  // Order matters: handle code spans first so we don't bold/italic content inside backticks.
+  return line
+    // `inline code` -> dim cyan
+    .replace(/`([^`]+)`/g, (_, code: string) => stylize(`${ANSI.cyan}${ANSI.dim}`, code))
+    // **bold** -> bold (after code so the ** in code isn't matched)
+    .replace(/\*\*([^*]+)\*\*/g, (_, b: string) => stylize(ANSI.bold, b))
+    // *italic* -> italic. Negative-lookahead/behind avoid ** false-matches and
+    // bare * in log content.
+    .replace(/(^|[^*])\*([^*\s][^*]*?)\*(?!\*)/g, (_, prefix: string, i: string) => `${prefix}${stylize(ANSI.italic, i)}`)
```

ESLint

[error] Unexpected non-greedy constant quantifier. The quantifier is effectively possessive, so it doesn't matter whether it is greedy or not.

(regexp/no-useless-lazy)